### PR TITLE
ROECCT-149 - Changes to Mongo DB

### PR DESCRIPTION
**Pull request by krenjith-ch**
### JIRA link

https://companieshouse.atlassian.net/browse/ROECCT-149

### Work checklist

- [ ] Tests added where applicable

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/RelevantStatementsType.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+
+public enum RelevantStatementsType {
+    CEASED_TO_BE_REGISTRABLE_BENEFICIAL_OWNER("ceased_to_be_registrable_beneficial_owner"),
+    NO_CEASED_TO_BE_REGISTRABLE_BENEFICIAL_OWNER("no_ceased_to_be_registrable_beneficial_owner"),
+    TRUST_INVOLVED_IN_THE_OE("trust_involved_in_the_oe"),
+    NO_TRUST_INVOLVED_IN_THE_OE("no_trust_involved_in_the_oe"),
+    BECOME_OR_CEASED_BENEFICIARY_OF_A_TRUST("become_or_ceased_beneficiary_of_a_trust"),
+    NO_BECOME_OR_CEASED_BENEFICIARY_OF_A_TRUST("not_become_or_ceased_beneficiary_of_a_trust");
```
Comment (ConnorOS-ch) [inline src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/RelevantStatementsType.java:11]: this says `"NOT_become_or_ceased_beneficiary_of_a_trust"` sould it be `no_become...`?

File: src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dao/UpdateDao.java [hunk 2]
```diff
@@ -45,7 +46,34 @@ public class UpdateDao {
 
     @Field("review_trusts")
     private List<TrustDataToReviewDao> reviewTrusts;
+    @Field("ceased_to_be_registrable_beneficial_owner")
+    private RelevantStatementsType relevantStatementsType;
+    @Field("trust_involved_in_the_oe")
+    private RelevantStatementsType relevantStatementsType2;
+    @Field("become_or_ceased_beneficiary_of_a_trust")
+    private RelevantStatementsType relevantStatementsType3;
```
Comment (ConnorOS-ch) [inline src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dao/UpdateDao.java:54]: Should these be the new names `ceased_to_be_registrable_beneficial_owner`, `trust_involved_in_the_oe`, `become_or_ceased_beneficiary_of_a_trust`

File: src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dto/UpdateDto.java [hunk 3]
```diff
@@ -29,6 +30,9 @@ public class UpdateDto {
     public static final String TRUST_DATA_FETCHED = "trust_data_fetched";
 
     public static final String REVIEW_TRUSTS_FIELD = "review_trusts";
+    private static final String RELEVANT_STATEMENT1 = "relevant_period_beneficial_owner";
+    private static final String RELEVANT_STATEMENT2 = "relevant_period_trust_involved";
+    private static final String RELEVANT_STATEMENT3 = "relevant_period_trust_beneficiary";
```
Comment (ConnorOS-ch) [inline src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dto/UpdateDto.java:35]: Should these be the new names `ceased_to_be_registrable_beneficial_owner`, `trust_involved_in_the_oe`, `become_or_ceased_beneficiary_of_a_trust`

File: src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/RelevantStatementsType.java (hunk 1 above)
Comment (krenjith-ch) [reply src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/RelevantStatementsType.java]: Updated, thanks

File: src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dto/UpdateDto.java (hunk 3 above)
Comment (krenjith-ch) [reply src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dto/UpdateDto.java]: Done, thanks

**Review by ConnorOS-ch — APPROVED** [review]
LGTM :)

**Pull request merged**